osf/exceptions.py

```diff
@@ -175,6 +175,13 @@ class PreviousSchemaResponseError(SchemaResponseError):
     pass
 
 
+class RegistrationBulkCreationContributorError(OSFError):
+    """Raised if contributor preparation has failed"""
+
+    def __init__(self, error=None):
+        self.error = error if error else 'Contributor preparation error'
```

Comment:
Unlike `RegistrationBulkCreationRowError` that contains lots of details for error reporting, this exception will be caught and re-thrown as ``RegistrationBulkCreationRowError`.